code/game/objects/structures/arclumin.dm

```diff
@@ -0,0 +1,21 @@
+/*
+
+At 7:30 AM on March 8th, 2017, ArcLumin died in a fatal car crash on impact.
```

Comment:
I dont mean the anonymous statue @Leshana, I was referring to the comment in particular